src/tools/pontoon/users.md

```diff
@@ -4,65 +4,96 @@
 
 ## Creating an account
 
-Pontoon currently uses Firefox Accounts to manage users, so you will need to [create an account](https://accounts.firefox.com/signup) in order to log in.
+While Pontoon can be configured to use [different login systems](https://mozilla-pontoon.readthedocs.io/en/latest/admin/deployment.html#environment-variables) (Django, GitHub, etc.), the Mozilla instance hosted at [pontoon.mozilla.org](https://pontoon.mozilla.org/) uses Firefox accounts.
 
-To create an account on [Pontoon](https://pontoon.mozilla.org/), simply click on the *Sign In* button in the top right corner.
-
-![Pontoon login](../../assets/images/pontoon/users/menu_login.png)
+To create an account, click on the `Sign In` button in the top right corner and follow the instructions.
 
 ## User settings
 
-[User settings](https://pontoon.mozilla.org/settings/) can be accessed by clicking the user avatar in the top right corner, and selecting *Settings*.
+User settings can be accessed by:
+* Clicking the user avatar in the top right corner, and selecting `Settings` from the dropdown menu.
+* Opening directly the `/settings` URL (e.g. [pontoon.mozilla.org/settings](https://pontoon.mozilla.org/settings/)).
+* Using the `CHANGE SETTINGS` button in the [Profile page](profile.md).
 
-![Pontoon login](../../assets/images/pontoon/users/menu_settings.png)
+![User menu with profile settings](../../assets/images/pontoon/users/profile_settings.png "User menu with profile settings")
 
-Profile picture can be set or updated on [Gravatar](http://gravatar.com/) by clicking directly on the image.
+### Profile picture
 
-In the *Personal information* section it’s possible to update the displayed name and change the email address used to log in to Pontoon.
+The profile picture can be set or updated on [Gravatar](http://gravatar.com/) by clicking the image displayed at the top of the settings page (when hovered, the picture will display an `Update profile picture` message).
 
-It’s also possible to set the default homepage displayed when visiting Pontoon as a logged in user. Current options are:
-* Default homepage (Localization page for the Pontoon Intro project).
-* [A Team page](teams_projects.md#team-page).
+### Personal information
 
-![Pontoon login](../../assets/images/pontoon/users/profile_page.png)
+In this section it’s possible to update the display name and other personal information:
+* `Display name`: it’s the name displayed in Pontoon, for example when submitting a translation or suggestion. It’s also used as the author name when committing translation to VCS (i.e. git or hg repositories).
+* `Username`: this is currently used to generate the URL for the [Profile page](profile.md) (e.g. `https://pontoon.mozilla.org/contributors/USERNAME/`). If not provided, an hash will be used instead of the username.
```

Comment:
```suggestion
* `Username`: this is currently used to generate the URL for the [Profile page](profile.md) (e.g. `https://pontoon.mozilla.org/contributors/USERNAME/`). If not provided, a hash will be used instead of the username.
```